src/CompositeNormL1.jl

```diff
@@ -0,0 +1,39 @@
+# Composition of the L1 norm with a function
+export CompositeNormL1
+
+@doc raw"""
+    CompositeNormL1(h,c!,J!,A,b)
+
+Returns the ``\ell_{1}`` norm operator composed with a function
+```math
+f(x) = λ ||c(x)||_1
```

Comment:
h is always NormL1 but it takes an additionnal argument $\tau$ such that $h= \tau \| \cdot \|_1$. Maybe it would be clearer to just add $\tau$ as an argument instead.

Reply:
Yes, but if you know the type `CompositeNormL1`, you know that `h = NormL1`, etc.